lib/core/datatable/data/data-column.model.ts

```diff
@@ -29,7 +29,7 @@ export interface DataColumnTypes {
 
 export type DataColumnType = keyof DataColumnTypes;
 
-export interface DataColumn {
+export interface DataColumn<CustomData = unknown> {
```

Comment:
traditionally this kind of generics is using `T`, also, not sure if that's not a breaking change for the apps that are using this interface...